tools/cppcheck/workflows/cppcheck-dashboard.yml

```
@@ -0,0 +1,341 @@
+      run: |
+        sudo apt-get install -y cmake ninja-build libgsl-dev
+        cmake -B build -G Ninja -DCMAKE_EXPORT_COMPILE_COMMANDS=ON || true
+    
+    - name: Run quick analysis
+      if: github.event_name == 'pull_request' || inputs.analysis_type == 'quick' || inputs.analysis_type == 'all'
+      run: |
+        python3 tools/cppcheck/scripts/analyze.py \
+          --profile quick_check \
+          --format json \
+          --incremental
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)